### PR TITLE
[Brno] Add next Pyvo

### PR DESCRIPTION
**Pull request by Dormouse759**

### CODE REVIEW

File: series/brno-pyvo/events/2019-06-27-maticove.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+topic: Maticové
+venue: artbar
+description: |
+    Co jsou to matice a na co se používají? Suchá teorie a nudná matematika,
+    nebo mohou i bavit?
+
+    Přijďte se podívat na přednášky a dozvědět se, co vlastně matice jsou, jak
+    se používají a na co jsou užitečné.
+    ---
+    What exactly matrices are and what are they used for? Plain theory and
```
Comment (encukou) [inline series/brno-pyvo/events/2019-06-27-maticove.yaml]:
```suggestion
    What are matrices, exactly, and what are they used for? Plain theory and
```

File: series/brno-pyvo/events/2019-06-27-maticove.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+        ---
+
+        V této přednášce Vám představím opensource aplikaci Blender a její
+        některé klíčové funkce a techniky. Bude zahrnuta mírná teorie opřená o
+        praktické ukázky
+
+        ---
+
+        In this lecture I will introduce you an opensource app Blender and some
```
Comment (encukou) [inline series/brno-pyvo/events/2019-06-27-maticove.yaml]:
```suggestion
        In this lecture I will introduce the opensource app Blender and some
```

File: series/brno-pyvo/events/2019-06-27-maticove.yaml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        of its key functions and techniques. It will include an easy theory
```
Comment (encukou) [inline series/brno-pyvo/events/2019-06-27-maticove.yaml]:
```suggestion
        of its key functions and techniques. It will include some easy theory
```

**Comment by Dormouse759** [comment]
Thanks for the review!

**Pull request merged**